lib/masksets.gd

```diff
@@ -0,0 +1,55 @@
+#############################################################################
+##
+#W  masksets.gd                  GAP library                     Steve Linton
+##
+##
+#Y  Copyright (C) 2017 The GAP Group
+##
+##  This file declares the operations for mask sets.
+##
+
+
+
+#############################################################################
+##
+#F  MakeMaskSet(<width>[, <width>[, <width>...]]])
+##  <#GAPDoc Label="MakeMaskSet">
+##  <ManSection>
+##  <Func Name="MakeMaskSet" Arg='width....'/>
+##
+##  <Description>
+##
+##  This function sets up the machinery for a set of bitfields of the given
+##  widths. The total of the widths must not exceed 60 bits on 64-bit architecture
+##  or 28 bits on a 32-bit architecture and the number of widths must not exceed
+##  10 on a 64-bit aerchitecture or 6 on a 32-bit architecture. Also for performance 
+##  reasons some checks that one might wish to do are ommitted. In particular, 
+## the builder and setter functions do not check if the value[s] passed to them are 
+## negative or too large.
```

Comment:
Also: I think this implicitly says: "the field values are non-negative values", but this is never said explicitly (or am I just overlooking it?)